src/app/core/top-panel/top-panel.component.html

```diff
@@ -12,12 +12,11 @@
     <div class="d-flex justify-content-end col order-lg-3">
       <div class="panel-item lead mr-2"
            *ngIf="isWorkspaceSet()">
-        <a [routerLink]="['/workspace']"
-           title="Choose workspace">
         <span>
-          {{workspace.projectCode}}@{{workspace.organizationCode}}
+          <a [routerLink]="['/workspace']" title="Choose workspace organization">{{workspace.organizationCode}}</a>
+          &nbsp;/&nbsp;
```

Comment:
Are these non-breaking spaces really useful here? I mean there are breaking spaces before and after them. Isn't that a problem? (I assume you want to prevent splitting this workspace location into two lines when the screen is small.)